Rultor release fails due to multiple test timeouts

Getting [this](https://github.com/objectionary/lints/issues/759#issuecomment-3352242442) during Rultor release:

```text
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   HomeNamesTest.placesHomeObjectsAsReserved(Path) » Timeout placesHomeObjectsAsReserved(java.nio.file.Path) timed out after 90 seconds
[ERROR]   LtByXslTest.validatesEoPacksForErrors » Timeout validatesEoPacksForErrors() timed out after 180 seconds
[ERROR]   SourceTest.lintsInMultipleThreads()[1] » Timeout lintsInMultipleThreads() timed out after 60 seconds
[ERROR]   SourceTest.lintsInMultipleThreads()[2] » Timeout lintsInMultipleThreads() timed out after 60 seconds
[ERROR]   SourceTest.returnsEmptyListOfDefects » Timeout returnsEmptyListOfDefects() timed out after 45 seconds
```

Let's fix it

--- a/src/test/groovy/org/eolang/lints/HomeNamesTest.groovy
+++ b/src/test/groovy/org/eolang/lints/HomeNamesTest.groovy
@@ -7,6 +7,7 @@ package org.eolang.lints
 import com.yegor256.MayBeSlow
 import java.nio.file.Path
 import java.nio.file.Paths
+import java.util.concurrent.TimeUnit
 import org.hamcrest.MatcherAssert
 import org.hamcrest.Matchers
 import org.junit.jupiter.api.Tag
@@ -24,6 +25,7 @@ import org.junit.jupiter.api.io.TempDir
 @ExtendWith(MayBeSlow)
 final class HomeNamesTest {
 
+  @Timeout(unit = TimeUnit.MINUTES, value = 3L)
   @Test
   void placesHomeObjectsAsReserved(@TempDir final Path temp) {
     final String csv = temp.resolve("reserved.csv").toString()
@@ -35,6 +37,7 @@ final class HomeNamesTest {
     )
   }
 
+  @Timeout(unit = TimeUnit.MINUTES, value = 3L)
   @Test
   void placesHomeObjectsWithCorrectNames(@TempDir final Path temp) {
     final String csv = temp.resolve("reserved.csv").toString()

--- a/src/test/java/org/eolang/lints/LtByXslTest.java
+++ b/src/test/java/org/eolang/lints/LtByXslTest.java
@@ -18,6 +18,7 @@ import java.util.Collection;
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 import java.util.function.Predicate;
 import java.util.stream.Collectors;
@@ -359,7 +360,7 @@ final class LtByXslTest {
 
     @SuppressWarnings("StreamResourceLeak")
     @Tag("deep")
-    @Timeout(180L)
+    @Timeout(unit = TimeUnit.MINUTES, value = 5L)
     @Test
     void validatesEoPacksForErrors() throws IOException {
         Files.walk(Paths.get("src/test/resources/org/eolang/lints/packs/single"))

--- a/src/test/java/org/eolang/lints/SourceTest.java
+++ b/src/test/java/org/eolang/lints/SourceTest.java
@@ -28,6 +28,7 @@ import java.util.Collection;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 import org.cactoos.bytes.BytesOf;
@@ -61,12 +62,17 @@ import org.objectweb.asm.Opcodes;
  * Test for {@link Source}.
  *
  * @since 0.0.1
+ * @todo #767:60min Decrease timeouts for source linting.
+ *  As for now, most lints are too slow, we need to optimize them first, so they
+ *  run in milliseconds, not seconds/minutes. It should decrease our build time too.
+ *  After that, we need to decrease our test timeouts. Don't forget to remove this puzzle.
  * @checkstyle MethodBodyCommentsCheck (50 lines)
  */
 @SuppressWarnings("PMD.TooManyMethods")
 @ExtendWith(MktmpResolver.class)
 final class SourceTest {
 
+    @Timeout(unit = TimeUnit.SECONDS, value = 60L)
     @Test
     void returnsEmptyListOfDefects() throws IOException {
         MatcherAssert.assertThat(
@@ -144,7 +150,7 @@ final class SourceTest {
     }
 
     @Tag("deep")
-    @Timeout(60L)
+    @Timeout(unit = TimeUnit.MINUTES, value = 2L)
     @RepeatedTest(2)
     void lintsInMultipleThreads() {
         MatcherAssert.assertThat(

